pkg/cmd/admin/policy/whatcanido.go

```diff
@@ -53,32 +80,85 @@ const (
 	tabwriterFlags    = 0
 )
 
-func (o *whatCanIDoOptions) complete(f *clientcmd.Factory, args []string) error {
-	if len(args) != 0 {
-		return errors.New("no arguments are supported")
+func (o *whatCanIDoOptions) Complete(f *clientcmd.Factory, args []string) error {
+	if o.ListAll && o.AllNamespaces {
+		return errors.New("--list and --all-namespaces are mutually exclusive")
+	}
+
+	switch len(args) {
+	case 3:
+		o.ResourceName = args[2]
+		fallthrough
+	case 2:
+		if o.ListAll {
```

Comment:
Nope, --all-namespaces "tests the action on all namespaces" (I guess those you have access to - make it cleaner in the flag help). So you need to fix the error message below.